bindings/c/src/lib.rs

```diff
@@ -301,6 +302,55 @@ pub unsafe extern "C" fn libsql_connect(
     0
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn libsql_load_extension(
+    conn: libsql_connection_t,
+    path: *const std::ffi::c_char,
+    entry_point: *const std::ffi::c_char,
+    out_err_msg: *mut *const std::ffi::c_char,
+) -> std::ffi::c_int {
+    if path.is_null() {
+        set_err_msg("Null path".to_string(), out_err_msg);
+        return 1;
+    }
+    let path = unsafe { std::ffi::CStr::from_ptr(path) };
+    let path = match path.to_str() {
+        Ok(path) => path,
+        Err(e) => {
+            set_err_msg(format!("Wrong path: {}", e), out_err_msg);
+            return 2;
+        }
+    };
+    let mut entry_point_option = None;
+    if !entry_point.is_null() {
+        let entry_point = unsafe { std::ffi::CStr::from_ptr(entry_point) };
+        entry_point_option = match entry_point.to_str() {
+            Ok(entry_point) => Some(entry_point),
+            Err(e) => {
+                set_err_msg(format!("Wrong entry point: {}", e), out_err_msg);
+                return 4;
```

Comment:
Do these 2/4/5/6 error code has special meaning? Or it's just some numbers to distinguish between different error causes?

Reply:
Just different values to make it easier to distinguish which code path caused the error.